trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk

```diff
@@ -1,39 +1,35 @@
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-
-{% extends "index.njk" %}
-
-{% set backLink = { href: "/utilisation-reports" } %}
-
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/radios/macro.njk' import govukRadios %}
+{% from 'govuk/components/input/macro.njk' import govukInput %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% extends 'index.njk' %}
+{% set backLink = { href: '/utilisation-reports' } %}
 {% block pageTitle %}
   Find reports by year
 {% endblock %}
 {% block content %}
-  <div class="govuk-grid-row govuk-!-padding-top-7">
-    <div
-      class="govuk-grid-column-full">
+  <div class='govuk-grid-row govuk-!-padding-top-7'>
+    <div class='govuk-grid-column-full'>
       {% if errorSummary | length %}
         {{ govukErrorSummary({
-          titleText: "There is a problem",
+          titleText: 'There is a problem',
```

Comment:
Do we need to change from double quote to single quote? Looks like some digression has been happening - when searching the nunjucks files:

- `text: "` = 706 results
- `text: '` = 67 results

Reply:
Definitely not for this PR but wondering if we have ever considered linting for nunjucks files? It's hard to infer the intended patterns for going forwards given how inconsistent the existing files are on things like quotes and spacing and depending on individual IDE setup it definitely suggests different things on spacing etc to others

Reply:
No idea why this change happened, but we definitely should look into adding better and more consistent prettier for nunjucks files